### DISCUSSION TITLE
Contribution Workflow

### DISCUSSION THREAD

**Discussion by Vishrut19** [idurar/idurar-erp-crm#216] [Ideas]
Hey @salahlalami!
I think the file `Contribution Workflow` needs a whole lot of improvement as it is very unclear and very misleading and confusing, rather creating a good `contribution workflow` guideline would attract more folks to contribute to this project

**Comment by salahlalami** [comment]
@Vishrut19  Thank you for your interest to contribute to IDURAR project ,
Here a Contribution Guide : [Contribution](https://github.com/idurar/idurar-erp-crm/blob/master/CONTRIBUTING.md)
